### PR TITLE
Matt experfwizscript

### PR DESCRIPTION
**Pull request by Canthv0**
**Issue:**
Moving Experfwiz from a module to a script

**Reason:**
Integrate experfwiz with css-exchange and get it into a script

**Validation:**
Tested and everything seems to be working

### CODE REVIEW

**Comment by dpaulson45** [comment]
`Get-ExPerfWiz.ps1` isn't saved as a BOM. Save it as that way and run `.build\CodeFormatter.ps1` to make sure everything else is good. 

Also create a docs page to be included as to how to run the script.

**Comment by bill-long** [comment]
I noticed you took out the auto-update check. We do have auto-update infrastructure for scripts in this repo. To use it, dot source Test-ScriptVersion and then call it. For example, SourceSideValidations uses it like this:

```powershell
. $PSScriptRoot\..\..\Shared\Test-ScriptVersion.ps1

if (Test-ScriptVersion -AutoUpdate) {
  # Update was downloaded, so stop here.
  Write-Host "Script was updated. Please rerun the command."
  return
}
```

Not needed for this PR, of course. Just figured you might want to look at keeping the auto-update ability, since we make it super easy.

**Comment by lusassl-msft** [comment]
@Canthv0 the job failed because `ExPerfWiz.md` was not added to `mkdocs.yml`

File: docs/Performance/ExPerfWiz.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+# About ExPerfWiz
+ExPerfWiz is a PowerShell based script to help automate the collection of performance data on Exchange 2013, 2016 and 2019 servers.  Supported operating systems are Windows 2008, 2008 R2, 2012, 2012 R2, 2016 and 2019 Core and Standard.
```
Comment (dpaulson45) [inline docs/Performance/ExPerfWiz.md:2]: Remove Windows 2008, and 2008 R2.

**Pull request merged**